base/commandLine.cpp

```diff
@@ -813,6 +817,7 @@ Common::String parseCommandLine(Common::StringMap &settings, int argc, const cha
 			DO_OPTION('g', "gfx-mode")
 			END_OPTION
 
+
```

Comment:
this change is not necessary

Reply:
Yes indeed, my apologies. I had to move the move the DO_LONG_OPTION functionality below the one that was created for the themepath (otherwise the default location of de shaders.dat file could not be found). I accidentally added an additional line feed. I will remove it.

Reply:
Should be fixed now.